test/e2e_node/remote/utils.go

```diff
@@ -43,6 +60,16 @@ func installCNI(host, workspace string) error {
 	if output, err := SSH(host, "sh", "-c", cmd); err != nil {
 		return fmt.Errorf("failed to install cni plugin on %q: %v output: %q", host, err, output)
 	}
+
```

Comment:
Add a comment here to explain:
The added cni network config is not needed for kubenet. It is only for testing cni network plugin. But place it there for consistency and simplicity.